src/main/java/uk/gov/ons/census/fwmt/rmadapter/service/impl/RMAdapterServiceImpl.java

```diff
@@ -41,5 +45,15 @@ public void sendJobRequest(ActionInstruction actionInstruction) throws GatewayEx
             .triggerEvent(actionInstruction.getActionCancel().getCaseId(), CANONICAL_CANCEL_FAILED, LocalTime.now());
       }
     }
+    else if (actionInstruction.getActionUpdate() != null){
+      if (!StringUtils.isEmpty(actionInstruction.getActionUpdate().getPause().getId())) {
+        jobServiceProducer.sendMessage(CanonicalJobHelper.newUpdateJob(actionInstruction));
+        gatewayEventManager
+            .triggerEvent(actionInstruction.getActionUpdate().getPause().getId(), CANONICAL_UPDATE_SENT, LocalTime.now());
```

Comment:
if its supposed to contain an id but doesn't throw exception, if it cannot match request to any of the types should also throw an exception

Reply:
Updated as discussed